### PR TITLE
Restructure guide "singing txs offline" - Closes #1315

### LINKED ISSUE #1315: Update offline transaction creation and signing docs
### What

Update transaction creation and signing offline https://liskhq.github.io/lisk-docs/lisk-sdk/guides/node-management/signing-transactions-offline.html#creating-and-signing-the-transaction

### How

- User should be able to create and sign transactions offline 
   - Provide docs for signed transactions in JSON format
   - Provide docs for signed transactions in hex format
   - Provide steps for send the transactions via
      - `lisk-core`/ bapp CLI
      - Lisk Service node
      - Lisk Core / bapp RPC endpoint

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1315

### CODE REVIEW

File: modules/ROOT/pages/guides/app-development/decoding-transactions-blocks.adoc [hunk 1]
```diff
@@ -41,13 +42,12 @@ const getClient = async () => {
 };
 ----
 
-== Decoding a block and all its' included transactions
+== Decoding & encoding blocks and their included transactions
 
-Write a simple script to decode the blocks.
-Create a new file `getBlockById.js`:
+The following script to decodes & encodes a block by a given ID.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/decoding-transactions-blocks.adoc]:
```suggestion
The following script shows how to decode & encode a block by a given ID.
```

File: modules/ROOT/pages/guides/app-development/decoding-transactions-blocks.adoc [hunk 2]
```diff
@@ -92,51 +98,60 @@ Assuming you wish to decode a block with the ID `52303f33684f5ffae4d307747db36d8
 node getBlockById.js 52303f33684f5ffae4d307747db36d86bde6e14d76958e197cb82eeab5ca6f92
 ----
 
-The decoded block object is then displayed in the terminal:
+The decoded block JSON will look similar to this:
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/decoding-transactions-blocks.adoc]:
```suggestion
The decoded block JSON will look similar to the following, as shown below:
```

File: modules/ROOT/pages/guides/app-development/decoding-transactions-blocks.adoc [hunk 3]
```diff
@@ -177,20 +198,28 @@ Assuming you wish to decode a transaction with the ID `130227fa63ac60edbbacb6dae
 node getTxById.js 130227fa63ac60edbbacb6dae709cf9304ab0181ef7ea28105764f6240d012f2
 ----
 
-The decoded transaction object is then returned to the terminal:
+The decoded transaction JSON will look similar to this:
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/decoding-transactions-blocks.adoc]:
```suggestion
The decoded transaction JSON will look similar to the following, as shown below:
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 4]
```diff
@@ -1,16 +1,26 @@
...
 :url_transfer_asset: references/lisk-framework/token-module.adoc#transferasset
 :url_register_multisig_asset: references/lisk-framework/keys-module.adoc#registerasset
 :url_register_delegate_asset: references/lisk-framework/dpos-module.adoc#registertransactionasset
 :url_vote_delegate_asset: references/lisk-framework/dpos-module.adoc#votetransactionasset
 :url_unlock_asset: references/lisk-framework/dpos-module.adoc#unlocktransactionasset
 :url_pom_asset: references/lisk-framework/dpos-module.adoc#pomtransactionasset
+:url_framework_httpapi: references/lisk-framework/http-api-plugin.adoc
+:url_service: lisk-service::index.adoc
 
 How to create and sign a transaction in a completely offline environment.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
How to create and sign a transaction in an entirely offline environment.
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 5]
```diff
@@ -1,16 +1,26 @@
...
 
-== Require the transaction and asset schema
+== Creating transactions
 
-To create, validate and sign transactions, it is necessary to access their schemas.
+=== Creating transactions with lisk-client
+
+==== Require the transaction and asset schema
+
+To create, validate and sign transactions offline with the lisk-client, it is necessary to access their schemas.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
To create, validate, and sign transactions offline with the lisk-client, it is necessary to access their schemas.
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 6]
```diff
@@ -1,16 +1,26 @@
...
 Create a new file `schemas.js` and add all required schemas in this file, like shown in the snippet below.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
Create a new file `schemas.js` and add all required schemas in this file, as shown in the snippet below.
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 7]
```diff
@@ -1,16 +1,26 @@
...
 
 In this example, we will send a Token Transfer transaction, therefore we need two different schemas:
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
In this example, we will send a Token Transfer transaction, therefore two different schemas are required:
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 8]
```diff
@@ -198,94 +208,94 @@ console.log(signedTransaction);
...
+    }
+  }
 }
-*/
 ----
-////
+====
 
-== Broadcasting a signed transaction
+== Broadcasting/ sending a signed transaction
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
== Broadcasting/sending a signed transaction
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 9]
```diff
@@ -198,94 +208,94 @@ console.log(signedTransaction);
...
+
+* <<via-api-client>>
+* <<via-rpc-api>>
+* <<via-cli>>
+* <<via-lisk-service-http-api>>
+* <<via-lisk-core-http-api>>
+
+[NOTE]
+====
+If the existing data needs to encoded/ decoded before posting, check the guide xref:{url_guides_decoding}[]
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
If the existing data needs to be encoded/decoded before posting, check the guide xref:{url_guides_decoding}[]
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 10]
```diff
@@ -320,3 +332,86 @@ getClient().then(async client => {
...
+[source,bash]
+----
+$ lisk-core transaction:send 0802100018022080c2d72f2a20e03c09bdc8c023d94cf66a5d352e6258380210d97d545abbf75668ea3736e3123229088094ebdc031214ab0041a7d3f7b2c290b5b834d46bdc7b7eb858151a0b73656e6420746f6b656e733a40faa2626d7306506b1999f48aa2f4b1ffdee01e641fa76d37a9d1d6fd8c225a81065c856ea625c52d138a7e3ba86b62913dc8e5aef8b5e307641ab66e0277a60b
+----
+
+=== Via Lisk Service HTTP API
+
+IMPORTANT: xref:{url_service}[Lisk Service] needs to be installed in oder to post a transaction via the Lisk Service HTTP API.
+
+cURL is one of the tools that can be used to send HTTP API requests to Lisk Service:
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
cURL is one of the tools that can be used to send HTTP API requests to Lisk Service as shown below:
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 11]
```diff
@@ -320,3 +332,86 @@ getClient().then(async client => {
...
+
+[source,bash]
+----
+curl -X POST -H "Content-Type: application/json" \
+-d '{ "transaction": "0802100018022080c2d72f2a20e03c09bdc8c023d94cf66a5d352e6258380210d97d545abbf75668ea3736e3123229088094ebdc031214ab0041a7d3f7b2c290b5b834d46bdc7b7eb858151a0b73656e6420746f6b656e733a40faa2626d7306506b1999f48aa2f4b1ffdee01e641fa76d37a9d1d6fd8c225a81065c856ea625c52d138a7e3ba86b62913dc8e5aef8b5e307641ab66e0277a60b"}' \
+"http://localhost:9901/api/v2/transactions"
+----
+
+The following response will be displayed, if the transaction was posted successfully.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
The following response shown below will be displayed if the transaction was posted successfully:
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 12]
```diff
@@ -320,3 +332,86 @@ getClient().then(async client => {
...
+
+----
+{"message":"Transaction payload was successfully passed to the network node","transactionId":"8a503843942e7d47ba0bef83fe735d26381f32a6ca6c96fb1cde902315f6220c"}
+----
+
+=== Via Lisk Core HTTP API
+
+IMPORTANT: The xref:{url_framework_httpapi}[] needs to be enabled on the node in oder to post a transaction via the node HTTP API.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
IMPORTANT: The xref:{url_framework_httpapi}[] needs to be enabled on the node in order to post a transaction via the node HTTP API.
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 13]
```diff
@@ -320,3 +332,86 @@ getClient().then(async client => {
...
+
+cURL is one of the tools that can be used to send HTTP API requests to Lisk Core:
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
cURL is one of the tools that can be used to send HTTP API requests to Lisk Core as shown below:
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 14]
```diff
@@ -320,3 +332,86 @@ getClient().then(async client => {
...
+
+[source,bash]
+----
+curl -X POST -H "Content-Type: application/json" \
+-d '{ "transaction": "0802100018022080c2d72f2a20e03c09bdc8c023d94cf66a5d352e6258380210d97d545abbf75668ea3736e3123229088094ebdc031214ab0041a7d3f7b2c290b5b834d46bdc7b7eb858151a0b73656e6420746f6b656e733a40faa2626d7306506b1999f48aa2f4b1ffdee01e641fa76d37a9d1d6fd8c225a81065c856ea625c52d138a7e3ba86b62913dc8e5aef8b5e307641ab66e0277a60b"}' \
+"http://localhost:4000/api/transactions"
+----
+
+The following response will be displayed, if the transaction was posted successfully.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
The following response shown below will be displayed if the transaction was posted successfully:
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a few minor points 👍

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc (hunk 11 above)
Comment (Tschakki) [reply modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
The following response will be displayed if the transaction was posted successfully:
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc (hunk 14 above)
Comment (Tschakki) [reply modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
The following response will be displayed if the transaction was posted successfully:
```

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 15]
```diff
@@ -320,3 +332,86 @@ getClient().then(async client => {
...
+=== Via CLI
+
+[source,bash]
+----
+$ lisk-core transaction:send 0802100018022080c2d72f2a20e03c09bdc8c023d94cf66a5d352e6258380210d97d545abbf75668ea3736e3123229088094ebdc031214ab0041a7d3f7b2c290b5b834d46bdc7b7eb858151a0b73656e6420746f6b656e733a40faa2626d7306506b1999f48aa2f4b1ffdee01e641fa76d37a9d1d6fd8c225a81065c856ea625c52d138a7e3ba86b62913dc8e5aef8b5e307641ab66e0277a60b
+----
+
+=== Via Lisk Service HTTP API
+
+IMPORTANT: xref:{url_service}[Lisk Service] needs to be installed in oder to post a transaction via the Lisk Service HTTP API.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
IMPORTANT: xref:{url_service}[Lisk Service] needs to be installed in order to post a transaction via the Lisk Service HTTP API.
```

**Review by chris529 — COMMENTED** [review]
Only 1 very small point 👍

**Review by ManuGowda — COMMENTED** [review]
@Tschakki I think there are group of audience who needs to create and sign transactions offline, they do not want to use lisk-client or run core or use service.

This is similar to command creating and signing transactions offline where it uses no client its complete offline. I am not sure if you intend to cover that, if this PR is just about restructuring then it looks all good.

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**